transformers/shared/constants/external.go

```diff
@@ -62,40 +64,58 @@ func GetTransformerContractNames(transformerLabel string) []string {
 	configKey := "exporter." + transformerLabel + ".contracts"
 	contracts := viper.GetStringSlice(configKey)
 	if len(contracts) == 0 {
-		log.Fatalf("No contracts configured for transformer: \"%v\"", transformerLabel)
+		logrus.Fatalf("No contracts configured for transformer: \"%v\"", transformerLabel)
 	}
 	return contracts
 }
 
-// Get the ABI for multiple contracts from config
+// GetABIFromContractsWithMatchingABI gets the ABI for multiple contracts from config
 // Makes sure the ABI matches for all, since a single transformer may run against many contracts.
-func GetContractsABI(contractNames []string) string {
+func GetABIFromContractsWithMatchingABI(contractNames []string) string {
 	initConfig()
 	if len(contractNames) < 1 {
```

Comment:
Does initConfig take time? You could assert before the initConfig to save that.

Reply:
I'm not sure whether it ever takes any significant amount of time, but it seems possible that it could - given that it's performing file reads. Could you elaborate on how you'd structure an assert? Wondering if we already achieve the same goal with the `initialized` var inside `initConfig()` (which flips to `true` the first time it's called and halts subsequent runs).

Reply:
What I mean is - couldn't you check len(contractNames) before calling init?